packages/python/plotly/plotly/io/_base_renderers.py

```diff
@@ -309,15 +293,7 @@ def activate(self):
         <script type="text/javascript">
         {win_config}
         {mathjax_config}
-        if (typeof require !== 'undefined') {{
-        require.undef("plotly");
-        define('plotly', function(require, exports, module) {{
-            {script}
-        }});
-        require(['plotly'], function(Plotly) {{
-            window._Plotly = Plotly;
-        }});
-        }}
+        {script}
```

Comment:
Also here similar to my comment https://github.com/plotly/plotly.py/pull/4763/files#r1757352938

You may add two scripts: one for configs; and the other to `import` plotly.js.

Reply:
I think this is actually embedding the entire plotly.js library so we wouldn't be including an `import` for plotly.js

Reply:
Yes. But I think if we could use ES6 import here there would be the benefit of having async loading.
Could you please give that a try?
To do that you need to add a separate `script` tag` after closing this script.
Thank you!

Reply:
If you scroll up in the code, this case is stated as follows:
```
# If not connected then we embed a copy of the plotly.js
# library in the notebook
```
So we should probably keep that in, right?